lib/server/createProvider.js

```diff
@@ -20,6 +20,18 @@ const createProvider = (sdkKey, user, options) => __awaiter(void 0, void 0, void
     }
     console.log('Initialized ld node client...');
     const flags = yield exports.ldClient.allFlagsState(user);
+    const flagsWithValues = flags.toJSON();
+    // After client is initialized, we loop through the flags and call the variation method on experiment flags.
+    yield exports.ldClient.waitForInitialization().then(() => {
+        Object.entries(flagsWithValues.$flagsState).forEach(([flagName, flagValue]) => {
+            var _a;
+            if ((_a = flagValue === null || flagValue === void 0 ? void 0 : flagValue.reason) === null || _a === void 0 ? void 0 : _a.inExperiment) {
+                // produces a feature event, which is necessary for a user to be considered a part of an experiment.
+                exports.ldClient.variation(flagName, user);
+            }
+        });
+    })
+        .catch((err) => console.error(err));
```

Comment:
Leaving this comment here to double-check that since this is a built file, that it's been built from the latest code in the other files!

Reply:
(at least, I think I remember that the files in `lib` were the ones that are built)